website/plugins/proto-messages/index.js

```diff
@@ -211,3 +260,46 @@ function getMatchingType(sourceDictionary, line) {
 
   return match;
 }
+
+const TLD = "(\\.\\w+)+";
+const LINK_ONLY = `https?\:\/\/.+${TLD}`;
+const LINK_WITH_TEXT = `\\[.+\\]\\(${LINK_ONLY}\\)`;
+
+const LINK_WITH_TEXT_SEPARATOR = "](";
+
+function getLinksFromALine(line) {
+  // Matches [any](any) or any://any.
+  const lineRegex = new RegExp(`${LINK_ONLY}|${LINK_WITH_TEXT}`, "g");
+  const matches = [];
+  let match = lineRegex.exec(line);
+
+  while (match) {
+    const separatorIndex = match[0].indexOf(LINK_WITH_TEXT_SEPARATOR);
```

Comment:
Optional check for safety, or index `0` is always there?

Reply:
I think index `0` is always there. If it's not there, then it will be `null` instead of an array (and it will not go to the loop). I'll add a comment for this. Thanks for pointing this out!